validator_api/test_time_inference.py

```diff
@@ -58,11 +58,12 @@ def parse_multiple_json(api_response):
             f"Invalid JSON object found in the response - field missing. The miner response was: {api_response}"
         )
         return None
+
     return parsed_objects
 
 
 async def make_api_call(
-    messages, model=None, is_final_answer: bool = False, use_miners: bool = True, target_uids: list[str] = None
+    messages, model=None, is_final_answer: bool = False, use_miners: bool = True, uids: list[str] | None = None
```

Comment:
```suggestion
    messages, model=None, is_final_answer: bool = False, use_miners: bool = True, uids: list[int] | None = None
```
You've typed uids as a list of strings here, but then as a list of ints later, it would be preferable to stick to one